bucket/smuxi.json

```
@@ -0,0 +1,23 @@
+{
+    "version": "1.1",
+    "description": "A free, distributed and user-friendly IRC / Twitter / XMPP / Campfire / JabbR client.",
+    "homepage": "https://smuxi.im/",
+    "license": "GPL-2.0-only",
+    "url": "https://smuxi.im/jaws/data/files/smuxi-1.1-win32-setup.exe#/dl.7z",
+    "hash": "sha1:67678e36736d2bace227922413b7a68a6bce971b",
+    "depends": "extras/gtk-sharp",
+    "shortcuts": [
+        [
+            "smuxi-frontend-gnome.exe",
+            "Smuxi"
+        ]
+    ],
+    "pre_install": "if (installed 'gtk-sharp' $global) {Copy-Item \"$(appdir gtk-sharp $global)\\current\\lib\\gtk-sharp-2.0\\*.dll\" \"$dir\\\" } else { error 'Please install `extras/gtk-sharp` first, then reinstall `extras/smuxi`.'}",
```

```suggestion
    "pre_install": [
        "if (installed 'gtk-sharp' $global) {",
        "    if ($architecture -eq '64bit') { $pg_files = ${Env:ProgramFiles(x86)} }",
        "    elseif ($architecture -eq '32bit') { $pg_files = $Env:ProgramFiles }",
        "    Copy-Item \"$pg_files\\GtkSharp\\*\\lib\\gtk-sharp-2.0\\*.dll\" \"$dir\\\"",
        "} else {",
        "    error 'Please install `extras/gtk-sharp` first, then reinstall `extras/smuxi`.'",
        "}"
    ],
```

The instllation path for `extras/gsharp` is actually under Program Files.